internal/renderers/femtovg/lib.rs

```diff
@@ -170,12 +170,16 @@ impl FemtoVGRenderer {
     }
 
     /// Render the scene using OpenGL.
-    pub fn render(&self) -> Result<(), i_slint_core::platform::PlatformError> {
+    pub fn render(
+        &self,
+        pre_present_callback: Option<Box<dyn FnOnce()>>,
+    ) -> Result<(), i_slint_core::platform::PlatformError> {
```

Comment:
Femtovg is public API since 1.3
So this is a breaking change.

I guess it should be `render_with_pre_present_callback`

Also it should be documented what that does.

Reply:
Gosh, you're right - I totally missed that it was public.